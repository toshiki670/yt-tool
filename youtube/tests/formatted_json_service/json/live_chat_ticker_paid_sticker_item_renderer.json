{
  "clickTrackingParams": "CAEQl98BIhMIz5X5ur27igMVoc80Bx03uTj5",
  "replayChatItemAction": {
    "actions": [
      {
        "clickTrackingParams": "CAEQl98BIhMIz5X5ur27igMVoc80Bx03uTj5",
        "addLiveChatTickerItemAction": {
          "item": {
            "liveChatTickerPaidStickerItemRenderer": {
              "id": "ChwKGkNPNk02SnlwdTRvREZaSEx3Z1FkNG84V0JR",
              "authorPhoto": {
                "thumbnails": [
                  {
                    "url": "https://yt4.ggpht.com/ytc/",
                    "width": 32,
                    "height": 32
                  },
                  {
                    "url": "https://yt4.ggpht.com/ytc/",
                    "width": 64,
                    "height": 64
                  }
                ],
                "accessibility": {
                  "accessibilityData": { "label": "ど投稿者名" }
                }
              },
              "startBackgroundColor": 4294953512,
              "endBackgroundColor": 4294947584,
              "durationSec": 300,
              "fullDurationSec": 300,
              "showItemEndpoint": {
                "clickTrackingParams": "COgBELHIBCITCM-V-bq9u4oDFaHPNAcdN7k4-Q==",
                "commandMetadata": {
                  "webCommandMetadata": { "ignoreNavigation": true }
                },
                "showLiveChatItemEndpoint": {
                  "renderer": {
                    "liveChatPaidStickerRenderer": {
                      "id": "ChwKGkNPNk02SnlwdTRvREZaSEx3Z1FkNG84V0JR",
                      "contextMenuEndpoint": {
                        "clickTrackingParams": "COoBEO-7BCITCM-V-bq9u4oDFaHPNAcdN7k4-Q==",
                        "commandMetadata": {
                          "webCommandMetadata": { "ignoreNavigation": true }
                        },
                        "liveChatItemContextMenuEndpoint": {
                          "params": "Q2g0S0hBb2FRMDgyVFRaS2VYQjFORzlFUmxwSVRIZG5VV1EwYnpoWFFsRWFLU29uQ2hoVlExcHNSRmg2UjI5dk4yUTBOR0ozWkU1UFlrWmhZMmNTQzBsb1NsY3RXVkJLTFhvd0lBRW9BVElhQ2hoVlEwbFRVbDlaVjFKdmRuZG9OVWQzYmxkWlJGZHViMEU0QWtnQlVCUSUzRA=="
                        }
                      },
                      "contextMenuAccessibility": {
                        "accessibilityData": { "label": "Chat actions" }
                      },
                      "timestampUsec": "1734868633176833",
                      "authorPhoto": {
                        "thumbnails": [
                          {
                            "url": "https://yt4.ggpht.com/ytc/AIdro_n8ow85lPIMwk-T8wuFdffYTGYYpo-uuN3qeV_iemmaslI=s32-c-k-c0x00ffffff-no-rj",
                            "width": 32,
                            "height": 32
                          },
                          {
                            "url": "https://yt4.ggpht.com/ytc/AIdro_n8ow85lPIMwk-T8wuFdffYTGYYpo-uuN3qeV_iemmaslI=s64-c-k-c0x00ffffff-no-rj",
                            "width": 64,
                            "height": 64
                          }
                        ]
                      },
                      "authorName": { "simpleText": "投稿者名" },
                      "authorExternalChannelId": "UCISR_YWRovwh5GwnWYDWnoA",
                      "timestampText": { "simpleText": "-2:44" },
                      "sticker": {
                        "thumbnails": [
                          {
                            "url": "//lh3.googleusercontent.com/1GF4XO0fhtEnQiPQwgLDQ49XhFOJxV7aJW3ku9eJEJptm1UwdE-vzQb4wTF5Utg5rcsSJuBY7sCkwyTLkeg=s104-rwa",
                            "width": 104,
                            "height": 104
                          },
                          {
                            "url": "//lh3.googleusercontent.com/1GF4XO0fhtEnQiPQwgLDQ49XhFOJxV7aJW3ku9eJEJptm1UwdE-vzQb4wTF5Utg5rcsSJuBY7sCkwyTLkeg=s208-rwa",
                            "width": 208,
                            "height": 208
                          }
                        ],
                        "accessibility": {
                          "accessibilityData": {
                            "label": "Shiba dog dressed in a traditional outfit waving Japanese fans"
                          }
                        }
                      },
                      "moneyChipBackgroundColor": 4294953512,
                      "moneyChipTextColor": 3741319168,
                      "purchaseAmountText": { "simpleText": "¥1,000" },
                      "stickerDisplayWidth": 104,
                      "stickerDisplayHeight": 104,
                      "backgroundColor": 4294953512,
                      "authorNameTextColor": 2315255808,
                      "trackingParams": "COoBEO-7BCITCM-V-bq9u4oDFaHPNAcdN7k4-Q==",
                      "isV2Style": true
                    }
                  },
                  "trackingParams": "COkBEI7RBiITCM-V-bq9u4oDFaHPNAcdN7k4-Q=="
                }
              },
              "authorExternalChannelId": "UCISR_YWRovwh5GwnWYDWnoA",
              "tickerThumbnails": [
                {
                  "thumbnails": [
                    {
                      "url": "//lh3.googleusercontent.com/R3GMKr3HlbWPhmhZkNZ1HKOdZIMKbhC5V6pqmG-",
                      "width": 32,
                      "height": 32
                    },
                    {
                      "url": "//lh3.googleusercontent.com/R3GMKr3HlbWPhmhZkNZ1HKOdZIMKbhC5V6pqmG-",
                      "width": 64,
                      "height": 64
                    }
                  ],
                  "accessibility": {
                    "accessibilityData": { "label": "Shiba Inu sticker pack" }
                  }
                }
              ],
              "trackingParams": "COgBELHIBCITCM-V-bq9u4oDFaHPNAcdN7k4-Q=="
            }
          },
          "durationSec": "300"
        }
      }
    ],
    "videoOffsetTimeMsec": "0"
  }
}
