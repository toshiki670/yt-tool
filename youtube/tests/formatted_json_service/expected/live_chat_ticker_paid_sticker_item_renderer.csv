id,authorExternalChannelId,postedAt,authorName,content,isModerator,membershipMonths,category
ChwKGkNPNk02SnlwdTRvREZaSEx3Z1FkNG84V0JR,UCISR_YWRovwh5GwnWYDWnoA,2024-12-22T20:57:13.176833+09:00,投稿者名,"¥1,000",false,,Ticker paid message item
